modes/info/evil-collection-info.el

```diff
@@ -44,6 +44,9 @@
     "h" 'evil-backward-char
     (kbd "<tab>") 'Info-next-reference
     (kbd "S-<tab>") 'Info-prev-reference
+    ;; This exists because tab isn't recognized as <tab> on terminal.
```

Comment:
Same here, I (and I guess @Ambrevar ) haven't had the time to fully digest these suggestions.

Reply:
We need something to look at and think about. That's why this pull request is here.

Reply:
I think that's a good idea we have something we can look at but you have one change (the RET key stuff) that's OK to merge and another change (TAB key / C-j/C-k/C-i) that will require some discussion.

Reply:
RET can wait. It's nothing urgent.